packages/@storefront/breadcrumbs/src/refinement-crumbs/index.ts

```diff
@@ -56,6 +56,8 @@ class RefinementCrumbs {
             index,
             field,
             range,
+            navigationLabel: navigation.label,
```

Comment:
You can still also do your logic for label and add it to the refinements state here. https://github.com/groupby/storefront/blob/4fa4f288c84b1c5a95dff742e3c05dc955127cba/packages/%40storefront/breadcrumbs/src/refinement-crumb/index.ts#L17-L23

Then you can use `props.label` in the `refinementCrumbs` template